src/app/features/psbt-signer/psbt-signer.context.ts

```diff
@@ -0,0 +1,36 @@
+import { createContext } from 'react';
+
+import { Money, createMoney } from '@shared/models/money.model';
+
+import { PsbtInput } from './hooks/use-parsed-inputs';
+import { PsbtOutput } from './hooks/use-parsed-outputs';
+
+export interface PsbtSignerContext {
+  accountInscriptionsBeingTransferred: string[];
+  accountInscriptionsBeingReceived: string[];
+  addressNativeSegwit: string;
+  addressTaproot: string;
+  addressNativeSegwitTotal: Money;
+  addressTaprootTotal: Money;
+  fee: Money;
+  isPsbtMutable: boolean;
+  psbtInputs: PsbtInput[];
+  psbtOutputs: PsbtOutput[];
+  shouldDefaultToAdvancedView: boolean;
+}
+
+export const psbtSignerContext = createContext<PsbtSignerContext>({
+  accountInscriptionsBeingTransferred: [],
+  accountInscriptionsBeingReceived: [],
+  addressNativeSegwit: '',
+  addressTaproot: '',
+  addressNativeSegwitTotal: createMoney(0, 'BTC'),
+  addressTaprootTotal: createMoney(0, 'BTC'),
+  fee: createMoney(0, 'BTC'),
+  isPsbtMutable: false,
+  psbtInputs: [],
+  psbtOutputs: [],
+  shouldDefaultToAdvancedView: false,
+});
```

Comment:
Is this legitimately not defined in many cases at start of flow, or is the mock data just to satisfy type?

If latter, suggest a default of `null`, a custom hook, and a `throw` if `null` so you can use `usePsbtSignerContext()` knowing data is defined